Build fails with fmt 8.1.1

Building with gcc 11.1 and fmt 8.1.1 fails with following message:
```
In file included from /usr/include/fmt/format.h:48,
                 from /tmp/nvbench/nvbench/detail/throw.cuh:21,
                 from /tmp/nvbench/nvbench/axes_metadata.cxx:21:
/usr/include/fmt/core.h: In instantiation of ‘constexpr fmt::v8::detail::value<Context> fmt::v8::detail::make_arg(T&&) [with bool IS_PACKED = true; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; fmt::v8::detail::type <anonymous> = fmt::v8::detail::type::custom_type; T = nvbench::axis_type&; typename std::enable_if<IS_PACKED, int>::type <anonymous> = 0]’:
/usr/include/fmt/core.h:1855:77:   required from ‘constexpr fmt::v8::format_arg_store<Context, Args>::format_arg_store(T&& ...) [with T = {std::basic_string_view<char, std::char_traits<char> >&, nvbench::axis_type&, nvbench::axis_type&}; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; Args = {std::basic_string_view<char, std::char_traits<char> >, nvbench::axis_type, nvbench::axis_type}]’
/usr/include/fmt/core.h:1872:38:   required from ‘constexpr fmt::v8::format_arg_store<Context, typename std::remove_cv<typename std::remove_reference<Args>::type>::type ...> fmt::v8::make_format_args(Args&& ...) [with Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; Args = {std::basic_string_view<char, std::char_traits<char> >&, nvbench::axis_type&, nvbench::axis_type&}]’
/usr/include/fmt/core.h:3119:44:   required from ‘std::string fmt::v8::format(fmt::v8::format_string<T ...>, T&& ...) [with T = {std::basic_string_view<char, std::char_traits<char> >&, nvbench::axis_type&, nvbench::axis_type}; std::string = std::__cxx11::basic_string<char>; fmt::v8::format_string<T ...> = fmt::v8::basic_format_string<char, std::basic_string_view<char, std::char_traits<char> >&, nvbench::axis_type&, nvbench::axis_type>]’
/tmp/nvbench/nvbench/axes_metadata.cxx:225:5:   required from here
/usr/include/fmt/core.h:1728:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1728 |       formattable,
      |       ^~~~~~~~~~~
/usr/include/fmt/core.h:1728:7: note: ‘formattable’ evaluates to false
```

--- a/nvbench/axes_metadata.cxx
+++ b/nvbench/axes_metadata.cxx
@@ -225,8 +225,8 @@ const axis_base &axes_metadata::get_axis(std::string_view name,
     NVBENCH_THROW(std::runtime_error,
                   "Axis '{}' type mismatch (expected {}, actual {}).",
                   name,
-                  type,
-                  axis.get_type());
+                  axis_type_to_string(type),
+                  axis.get_type_as_string());
   }
   return axis;
 }
@@ -240,8 +240,8 @@ axis_base &axes_metadata::get_axis(std::string_view name,
     NVBENCH_THROW(std::runtime_error,
                   "Axis '{}' type mismatch (expected {}, actual {}).",
                   name,
-                  type,
-                  axis.get_type());
+                  axis_type_to_string(type),
+                  axis.get_type_as_string());
   }
   return axis;
 }

